retriever/lib/fetch.py

```diff
@@ -0,0 +1,30 @@
+
+import os
+import sqlite3
+import pandas as pd
+from retriever.engines import sqlite
+from retriever.lib.defaults import HOME_DIR
+from retriever.lib.install import install_sqlite
+from retriever.lib.download import download
+
+
+
+def fetch(dataset):
+    """Install dataset into pandas dataframes.
+       each file in a separate dataframe
+    """
```

Comment:
Since this is a user facing function we want the docstring to be as descriptive as possible. I'd suggest something like:

```
"""Import a dataset into pandas data frames

Imports each table in the dataset into a data frame and stores the data frames
together in a list.

Parameters
----------
dataset : str
    Name of dataset

Returns
-------
list
    List of data frames, one for each table in the dataset

Example
-------
surveys, species, plots = rt.fetch('portal')
surveys.head()

"""
```

This is beyond what we have for some of our other functions, but it's good to start using the best docstrings possible as we add new functionality.